pdns/libssl.cc

```diff
@@ -678,6 +678,12 @@ std::unique_ptr<SSL_CTX, void(*)(SSL_CTX*)> libssl_init_server_context(const TLS
 #endif /* SSL_OP_PRIORITIZE_CHACHA */
   }
 
+  if (!config.d_enableRenegotiation) {
+#ifdef SSL_OP_NO_RENEGOTIATION
+    sslOptions |= SSL_OP_NO_RENEGOTIATION;
+#endif
```

Comment:
older openssl does not have `SSL_OP_NO_RENEGOTIATION` but does have `SSL_OP_NO_CLIENT_RENEGOTIATION`

Reply:
> older openssl does not have `SSL_OP_NO_RENEGOTIATION` but does have `SSL_OP_NO_CLIENT_RENEGOTIATION`

correction, "older openssl" should be libressl...

Reply:
Pushed an untested fix to support `SSL_OP_NO_CLIENT_RENEGOTIATION` when available.

Reply:
It looks good, but I wil test it tomorrow